Mailbox2.ino

```diff
@@ -176,4 +188,11 @@ void loop() {
   // TODO - add temperatur sensor and humitidy sensor, send data via MQTT.
   // Because everybody needs to know what the temperature is in their mailbox.
 
+  // Temperature reading
+  const float temp = float(digitalRead(tempSensorInPin)); // interpret input bits as float
```

Comment:
I will be using the breakout board version of the DHT22 with 3.3v, GND and Data, by the way, not the DHT22 by itself.